lib/CL/devices/level0/pocl-level0.cc

```diff
@@ -1393,7 +1393,7 @@ cl_int pocl_level0_set_kernel_exec_info_ext(
 
   assert(Kernel->data[ProgramDeviceI] != nullptr);
   Level0Kernel *L0Kernel = (Level0Kernel *)Kernel->data[ProgramDeviceI];
-
+  assert(L0Kernel != NULL);
```

Comment:
```suggestion
  assert(L0Kernel != nullptr);
```

Using the NULL macro in C++ code is considered non-idiomatic AFAIK.